V1/Services/OtpService.cs

```diff
@@ -103,9 +98,6 @@ private async Task<long> SaveOtpAsync(string mobileNumber, int code, int expiryT
         //TODO: Improve this
         public async Task<OtpConfirmationResponse> ConfirmOtpAsync(RequestOtpConfirmation payload, string authToken)
         {
-            if (payload.TestResult != null && !payload.isValid())
-                throw new ValidationException(Messages.Token_InvaldPayload);
```

Comment:
Token_InvalidPayload*, not sure if this is even used anywhere else but

Reply:
I moved the validation to the test result service for precaution, thanks